### PR TITLE
made antarctica free and without banner

### PR DESCRIPTION
**Pull request by nnngrach**
https://github.com/osmandapp/OsmAnd-Issues/issues/166

### CODE REVIEW

File: Sources/Controllers/Resources/OAManageResourcesViewController.mm [hunk 1]
```diff
@@ -425,7 +425,7 @@ - (void) willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOri
 
 - (BOOL) shouldHideBanner
 {
-    return _currentScope == kLocalResourcesScope || _iapHelper.subscribedToLiveUpdates || (self.region == _app.worldRegion && [_iapHelper isAnyMapPurchased]) || (self.region != _app.worldRegion && [self.region isInPurchasedArea]) || [self.region.regionId isEqualToString:_otherRegionId];
+    return _currentScope == kLocalResourcesScope || _iapHelper.subscribedToLiveUpdates || (self.region == _app.worldRegion && [_iapHelper isAnyMapPurchased]) || (self.region != _app.worldRegion && [self.region isInPurchasedArea]) || [self.region.regionId isEqualToString:_otherRegionId] || [self.region.regionId hasPrefix:OsmAnd::WorldRegions::AntarcticaRegionId.toNSString()];
```
Comment (PaulStets) [inline Sources/Controllers/Resources/OAManageResourcesViewController.mm:428]: Doesn't seem to be right. Maybe it would be better to make the product free and check for that while deciding whether to show the banner or not

File: Sources/Helpers/OAResourcesInstaller.mm [hunk 2]
```diff
@@ -19,6 +19,8 @@
 #import "OAResourcesUIHelper.h"
 #import "OAIAPHelper.h"
 
+#include <OsmAndCore/WorldRegions.h>
```
Comment (PaulStets) [inline Sources/Helpers/OAResourcesInstaller.mm:22]: Remove unused import

File: Sources/OAWorldRegion.mm [hunk 3]
```diff
@@ -247,12 +247,11 @@ - (void) setSuperregion:(OAWorldRegion *)superregion
 
 - (OAWorldRegion *) getPrimarySuperregion
 {
-    OAWorldRegion *primarySuperregion = _superregion;
```
Comment (PaulStets) [inline Sources/OAWorldRegion.mm:250]: Completely unnecessary change, revert it

File: Sources/Controllers/Resources/OAManageResourcesViewController.mm [hunk 4]
```diff
@@ -425,7 +425,7 @@ - (void) willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOri
 
 - (BOOL) shouldHideBanner
 {
-    return _currentScope == kLocalResourcesScope || _iapHelper.subscribedToLiveUpdates || (self.region == _app.worldRegion && [_iapHelper isAnyMapPurchased]) || (self.region != _app.worldRegion && [self.region isInPurchasedArea]) || [self.region.regionId isEqualToString:_otherRegionId];
+    return _currentScope == kLocalResourcesScope || _iapHelper.subscribedToLiveUpdates || (self.region == _app.worldRegion && [_iapHelper isAnyMapPurchased]) || (self.region != _app.worldRegion && [self.region isInPurchasedArea]) || [self.region.regionId isEqualToString:_otherRegionId] || [[self.region getPrimarySuperregion] getProduct].free;
```
Comment (alex-osm) [inline Sources/Controllers/Resources/OAManageResourcesViewController.mm:428]: Modify https://github.com/osmandapp/OsmAnd-iOS/blob/master/Sources/OAWorldRegion.mm#L489 method instead

File: Sources/Purchases/OAProducts.mm [hunk 5]
```diff
@@ -1680,7 +1676,8 @@ - (instancetype) init
                             self.trackRecording,
                             self.parking,
                             self.tripPlanning,
-                            self.osmEditing];
+                            self.osmEditing,
+                            self.antarctica];
```
Comment (alex-osm) [inline Sources/Purchases/OAProducts.mm:1680]: Antarctica still present at the list of purchases

File: Sources/Purchases/OAProducts.mm [hunk 6]
```diff
@@ -1468,14 +1468,10 @@ @implementation OAAntarcticaProduct
 - (instancetype) init
 {
     self = [super initWithIdentifier:kInAppId_Region_Antarctica];
+    self.free = YES;
```
Comment (alex-osm) [inline Sources/Purchases/OAProducts.mm:1471]: Is it needed?
